scripts/exregional_run_analysis.sh

```diff
@@ -356,27 +358,34 @@ rm coupler.res.newY coupler.res.newM coupler.res.newD
 #
 #-----------------------------------------------------------------------
 
-obs_file=${OBSPATH}/${YYYYMMDDHH}.rap.t${HH}z.prepbufr.tm00
-print_info_msg "$VERBOSE" "obsfile is $obs_file"
-if [ -r "${obs_file}" ]; then
-   cp_vrfy "${obs_file}" "prepbufr"
-else
-   print_info_msg "$VERBOSE" "Warning: ${obs_file} does not exist!"
-fi
+#obs_file=${OBSPATH}/${YYYYMMDDHH}.rap.t${HH}z.prepbufr.tm00
+#print_info_msg "$VERBOSE" "obsfile is $obs_file"
+#if [ -r "${obs_file}" ]; then
+#   cp_vrfy "${obs_file}" "prepbufr"
+#else
+#   print_info_msg "$VERBOSE" "Warning: ${obs_file} does not exist!"
+#fi
```

Comment:
Please remove commented code. This chunk will already be in git logs, so is recoverable.

Reply:
Done